README.md

```diff
@@ -218,18 +226,31 @@ and a back-end component written in Python.
 The front-end is compiled to Javascript during the build process
 and is distributed as static assets along with the Python package.
 
-_Note: Package versions are not prefixed with the letter `v`. You will need to disable this._
-
-```console
-$ jlpm config set version-tag-prefix ""
-```
-
 ### Release process
 
-This requires `node`, `build`, and `twine` to be installed.
+This requires `node`, `build`, `hatch` and `twine` to be installed.
 
 ```bash
-jlpm version [--major|--minor|--patch]  # updates package.json and creates git commit and tag
+# To set version (e.g. 7.0.0). hatch will update version string in package.json 
+hatch version "7.0.0"
```

Comment:
I don't love that this rewrites the `package.json`, but I suppose it will mostly just cause a large diff the first time. Thanks for updating the build docs!

Reply:
Well, it will update _only_ the version in `package.json`. We leverage the `hatch version` command to bump the version in `package.json` instead of doing it manually. I am not sure what do you mean by large diff.

Reply:
What I meant was that the hatch-nodejs-version plugin re-orders a bunch of the keys of the `package.json` when bumping the version (at least it did when I tried it yesterday). But that will only really produce a large diff on the first time we use it. I don't think it's a big deal, just a minor annoyance.